Solutions/Windows Security Events/Analytic Rules/NonDCActiveDirectoryReplication.yaml

```diff
@@ -42,19 +42,31 @@ query: |
   )
   on $left.SubjectLogonId == $right.TargetLogonId
   | project-reorder TimeGenerated, Computer, Account, IpAddress
-  | extend timestamp = TimeGenerated, AccountCustomEntity = Account, HostCustomEntity = Computer, SourceAddress = IpAddress 
+  | extend timestamp = TimeGenerated, SourceAddress = IpAddress 
```

Comment:
This line should be removed and the entity mapping for IP should be the original name - IpAddress.